packages/@secretlint/secretlint-rule-azure/README.md

```diff
@@ -0,0 +1,91 @@
+# @secretlint/secretlint-rule-azure
+
+A secretlint rule for Azure (i.e. Azure AD) secrets.
+
+## Install
+
+Install with [npm](https://www.npmjs.com/):
+
+    npm install @secretlint/secretlint-rule-azure
+
+## Usage
+
+Via `.secretlintrc.json`
+
+```json
+{
+    "rules": [
+        {
+            "id": "@secretlint/secretlint-rule-azure"
+        }
+    ]
+}
+```
+
+## MessageIDs
+
+### AzureTenantId
+
+> found Azure AD tenant ID: {{ID}}
+
```

Comment:
Can you add short description to each message Ids?

Or, just add a link to [EnvironmentCredential Class (Azure.Identity) - Azure for .NET Developers | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential).

Secretlint Lint error show the link to here.

Reply:
Something like this, perhaps? Also added some general links at the top.

Reply:
Nice!